generators/contract/templates/java/settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = '<%= name %>contract'
```

Comment:
The project name should just be `<%= name %>` - why are we appending `contract`?